numpy/core/setup_common.py

```diff
@@ -110,6 +110,7 @@ def check_api_version(apiversion, codegen_dir):
                 "xmmintrin.h",  # SSE
                 "emmintrin.h",  # SSE2
                 "immintrin.h",  # AVX
+                "altivec.h",  # VSX
```

Comment:
Did this used to be available on macOS too?

Reply:
Hi @mattip, this line generates the macro ``#define HAVE_ALTIVEC_H 1`` in the source file ``numpy/core/include/numpy/config.h``, which means that, if I remove it the optimizations of this PR wouldn't be generated for VSX/Power.